openlibrary/plugins/worksearch/code.py

```diff
@@ -1034,7 +1037,15 @@ class search_json(delegate.page):
     encoding = "json"
 
     def GET(self):
-        i = web.input()
+        i = web.input(author_key=[],
```

Comment:
To avoid duplicating this list, let's put it in a const variable; something like:

```py
# These URL params are made available as eg `&author_key=...` in the /search and /search.json endpoints
# The get translated to a proper solr query later.
SEARCH_API_FACET_PARAMS = { author_key=[], ... }
```

Then here (and in the other spot) we'd have:

```py
i = web.input(**SEARCH_API_FACET_PARAMS)
```

Reply:
Actually, I think that might introduce a subtle bug because those `[]` will be shared amongst all calls; this might be better if it works:

```py
SEARCH_API_FACET_PARAMS = { author_key=None, ... }
```